docs/guides/machines.md

```diff
@@ -1,54 +1,55 @@
-# Machines
+# Creating Machines
 
-A **state machine** is a finite set of [states](./statenodes.md) that can transition to each other deterministically due to events. A **statechart** is an extension of state machines; mainly, they can have:
+## createMachine
 
-- [Hierarchical](./hierarchical.md) (or nested) states
-- [Orthogonal](./parallel.md) (or parallel) states
-- [History](./history.md) states
-- And more, as described in [Statecharts: a Visual Formalism for Complex Systems](https://www.sciencedirect.com/science/article/pii/0167642387900359/pdf).
-
-## Configuration
-
-State machines and statecharts alike are defined using the `createMachine()` factory function:
+You can create a state machine with `createMachine`:
```

Comment:
(After my change specifying “a `createMachine` function:”): Should I specify “factory function”? Could we link it to a resource that teaches folks who have pretty basic JS skills (aka folks like me) the purpose of a factory function?